src/test/java/study/SetStudyTest.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetStudyTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.")
+    @Test
+    void checkSize() {
+        final int setSize = numbers.size();
+        int expectedSize = 3;
+
+        assertThat(setSize).isEqualTo(expectedSize);
+    }
```

Comment:
```suggestion
    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.")
    @Test
    void checkSize() {
        assertThat(numbers).hasSize(3);
    }
```
assertj 구문을 잘 살펴보면 요런 것도 가능합니다 😉